.atomist/handlers/event/Issue.ts

```diff
@@ -7,6 +7,7 @@ import { EventHandler, Tags } from '@atomist/rug/operations/Decorators'
 class OpenedIssue implements HandleEvent<GraphNode, GraphNode> {
     handle(event: Match<GraphNode, GraphNode>): Message | Plan {
         let issue = event.root() as any
+        console.log(issue);
```

Comment:
This issue is a SafeCommittingProxy that can be seen in the log when running the test. It is not interoperating well with the issue that is being created in the test.